src/main/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositApp.scala

```diff
@@ -38,6 +38,7 @@ class SplitMultiDepositApp(formats: Set[String], ldap: Ldap, permissions: Deposi
   private val fileMetadata = new AddFileMetadataToDeposit()
   private val depositProperties = new AddPropertiesToDeposit()
   private val setPermissions = new SetDepositPermissions(permissions)
+  private val reportDatasets = new ReportDatasets()
```

Comment:
I don't expect verbs as class names.

Reply:
I understand what you mean. If you have better suggestions for naming (@lindareijnhoudt also invited 😉), let me know.
For now, my thought was to keep naming in line with the other 'actions'. As you probably know, we previously had all these classes (`AddFileMetadata`, `AddPropertiesToDeposit`, `SetDepositPermissions`, etc.) extend from a class called `Action`, hence the class names describe the action being performed. Lately I removed the `Action` class and made it work with a for-comprehension instead, to make it more conform to the other projects. Still, though, the classes have there 'describe-the-action-you-perform' names.
This is also why I came up with a verb in the class name for `ReportDatasets`.